Source/Serilog.Exceptions.Refit/Destructurers/ApiExceptionDestructurer.cs

```diff
@@ -0,0 +1,61 @@
+namespace Serilog.Exceptions.Refit.Destructurers
+{
+    using System;
+    using System.Collections.Generic;
+    using global::Refit;
+    using Serilog.Exceptions.Core;
+    using Serilog.Exceptions.Destructurers;
+
+    /// <summary>
+    /// A destructurer for the Refit <see cref="ApiException"/>.
+    /// </summary>
+    /// <seealso cref="ExceptionDestructurer" />
+    public class ApiExceptionDestructurer : ExceptionDestructurer
+    {
+        private readonly bool destructureCommonExceptionProperties = true;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ApiExceptionDestructurer"/> class.
+        /// </summary>
+        /// <param name="destructureCommonExceptionProperties">Destructure common public Exception properties or not.</param>
+        public ApiExceptionDestructurer(bool destructureCommonExceptionProperties = true) => this.destructureCommonExceptionProperties = destructureCommonExceptionProperties;
```

Comment:
The influence of the parameter is not tested. Please add appropriate tests for completeness.

Reply:
Tests added